src/cpp/server/server_builder.cc

```diff
@@ -133,7 +135,7 @@ std::unique_ptr<Server> ServerBuilder::BuildAndStart() {
   // If the server has atleast one sync methods, we know that this is a Sync
   // server or a Hybrid server and the completion queue (server->cq_) would be
   // frequently polled.
-  int num_frequently_polled_cqs = (thread_pool != nullptr) ? 1 : 0;
+  int num_frequently_polled_cqs = has_sync_methods ? 1 : 0;
```

Comment:
Looks like we also need to add `has_sync_methods = true` after line 121 above. (This was a case, I missed earlier. The 'plugins' stuff was new.
```
   if (thread_pool == nullptr) {
     for (auto plugin = plugins_.begin(); plugin != plugins_.end(); plugin++) {
       if ((*plugin).second->has_sync_methods()) {
         thread_pool.reset(CreateDefaultThreadPool());
>>>>> Right here, we need to add  "has_sync_methods = true"
         break;
       }
     }